doc/developers/contributing.rst

```diff
@@ -952,6 +952,22 @@ The easiest and recommended way to accomplish this is to
 All logic behind estimator parameters,
 like translating string arguments into functions, should be done in ``fit``.
 
+Parameters can be passed to the ``fit`` method. Generally, these parameters
```

Comment:
I would say "why it is possible to pass parameter to the ``fit`` method, these should be ``` and also explain why (pipelining, grid-search). Also, the shape can be anything as long as the first dimension in n_samples.